cypress/e2e/sundial.cy.js

```diff
@@ -1,6 +1,6 @@
 describe("Sundial", () => {
   it("visits sundial.html", () => {
-    cy.visit("./sundial.html");
+    cy.visit("./.html");
```

Comment:
Ah yeah, the sundial tests aren't running because you might have accidentally deleted the sundial file name from this line here. If you do `cy.visit("./sundial.html");` like in the previous version, the tests should run again